src/signals/my-incidents/types.ts

```diff
@@ -6,9 +6,7 @@ export interface MyIncidentsValue {
   email?: string
   setEmail: (email: string) => void
   incidentsList?: MyIncident[]
-  incidentsDetail?: MyIncident
   setIncidentsList: (incidentsList: MyIncident[]) => void
```

Comment:
The types here are a bit misleading since `Location` and `sia:attachments` are not optional. Which also leads to extra code. I would suggest creating a separate type for MyIncidentDetail which extends the base one. Example: 

<img width="572" alt="Screenshot 2022-11-28 at 12 48 56" src="https://user-images.githubusercontent.com/46756331/204270621-258bb9b1-8897-47fb-9210-79139725894e.png">

Reply:
done